ftl/common/builder.py

```diff
@@ -102,6 +102,7 @@ def __init__(self, ctx, cache_namespace, args, descriptor_files):
         cache_repo = args.cache_repository
         if not cache_repo:
             cache_repo = self._target_image.as_repository()
+        use_minimum_ttl = ftl_util.use_minimum_ttl(descriptor_files, ctx)
```

Comment:
Maybe rename this to something that explains the why, not the what. `is_lock_file` or something similar.